cms/static/js/views/active_video_upload_list.js

```diff
@@ -22,10 +33,15 @@ define(
                 if (options.uploadButton) {
                     options.uploadButton.click(this.chooseFile.bind(this));
                 }
+                // error message modal for file uploads
+                this.fileErrorMsg = null;
```

Comment:
we are setting this to `null` at two places. I wonder if we really need this variable?